h5p-bildetema/src/components/LanguageSelector/LanguageSelector.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styles from "./LanguageSelector.module.scss";
+import type { Language as LanguageType } from "../../../../common/types/types";
+import { LanguageCode } from "../../../../common/types/LanguageCode";
+import { Language } from "..";
+
+type LanguageSelectorProps = {
+  languages: LanguageType[];
+  handleChange: (isFavorite: boolean, languageCode: LanguageCode) => void;
+};
+
+export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
+  languages,
+  handleChange,
+}) => {
+  const getAmountOfRows = (): number => {
+    if (languages.length <= 1) return 1;
+    console.info(languages.length / 2);
+
+    return Math.ceil(languages.length / 2);
+  };
+
+  return (
+    <div
+      className={styles.languageSelector}
+      style={{ gridTemplateRows: `repeat(${getAmountOfRows()}, 2rem)` }}
+    >
```

Comment:
If we can change the grid structure, this can be cleaned up.

```suggestion
  return (
    <div className={styles.languageSelector}>
```